src/interface/src/app/home/plan-table/plan-table.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="root-container">
   <h1>
-    {{ datasource.data.length && (loggedIn$ | async) ? 'Welcome back!' : 'Welcome to Planscape!' }}
+    {{ datasource.data.length && (loggedIn$ | async) ? 'Welcome back!' : login_enabled ? 'Welcome to Planscape!' : 'Welcome to Planscape early preview!' }}
```

Comment:
Ideally we shouldn't say "early" - it's superfluous.  I checked w/ Michele, and it's actually ok to keep this as just Welcome to Planscape.

Reply:
Should I change all references to "Planscape early preview" to just "Planscape preview"? (i.e the second header in the home page text)

Reply:
Also, I wouldn't branch this on login_enabled; when we release login, it might be considered a preview, similar to gmail being in "beta" for so many years.